lib/puppet/provider/x509_cert/openssl.rb

```diff
@@ -80,6 +80,8 @@ def create
         options << ['-CAcreateserial']
         options << ['-CA', resource[:ca]]
         options << ['-CAkey', resource[:cakey]]
+      else
+        options << ['-key', resource[:private_key]]
```

Comment:
In 268ae1ce1113e6d421819896f7c4ca90eac0da35 the `-signkey` option was added, which is an alias for `-key`. So I think this is now redundant